src/neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -134,6 +134,7 @@ private bool BlockAccount(ApplicationEngine engine, UInt160 account)
             StorageItem storage = engine.Snapshot.Storages.GetOrAdd(key, () => new StorageItem(new byte[1]));
             List<UInt160> accounts = storage.GetSerializableList<UInt160>();
             if (accounts.Contains(account)) return false;
+            if ((accounts.Count + 1) > StorageItem.MaxInteropEntries) throw new ArgumentException("Maximum number of blocked accounts exceeded");
```

Comment:
Blocking only 32 accounts may not be enough. We can divide them into multiple records.

Reply:
Also we can add two arguments to `GetBlockedAccounts` (offset, and count) and increase this to `4096/20` entries

Reply:
`GetBlockedAccounts` might return an iterator if we're concerned about the size of the result, but maybe it'd be easier for smart contracts to use `IsAccountBlocked(uint160)` call, it seems to be a more common use case rather than getting/iterating over whole blocked list.

Reply:
Then.. we need to change from list to key/value, because otherwise you will have a big list in memory, and the VM only will know that you have one stack item, the iterator. Please take a look to https://github.com/neo-project/neo/pull/1988/commits/94966f6c3fbd18741e0cc2b199fb5d786df3f5dc

Reply:
Since we have `IsAnyAccountBlocked()`, I think we don't need `GetBlockedAccounts()` any more.